### PR TITLE
Forkbranch

### PR DESCRIPTION
**Pull request by SeanChiu9418**
there is  a typo

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -6,4 +6,4 @@ Merge 3
 add Test2
 add Teest3
 Fort partice
-Fork_branch
+Fork_branch  typo
```
Comment (SeanChiuNKG) [inline README.txt]: I believe there is a typo

Comment (SeanChiuNKG) [reply README.txt]: please change it

**Comment by SeanChiuNKG** [comment]
please change it

File: README.txt (hunk 1 above)
Comment (SeanChiu9418) [reply README.txt]: OK I will

Comment (SeanChiu9418) [reply README.txt]: I did

**Comment by SeanChiu9418** [comment]
Check it

**Comment by SeanChiuNKG** [comment]
OK~

**Comment by SeanChiu9418** [comment]
Thanks

**Pull request merged**